### PR TITLE
Add everviz testing to PRs

### LINKED ISSUE #11041: We should run some basic checks on everviz on PRs to make sure it still works
We should run basic everviz checks on PRs to make sure it stays alive, they should not be required to pass, but if they fail, we are expected to follow it up with a PR so it does not stay red. Our goal is to remove the need to everviz, so this should hopefully not be long lived.

### PR DESCRIPTION
**Pull request by frode-aarstad**
**Issue**
Resolves #11041 

**Approach**

(Screenshot of new behavior in GUI if applicable)

- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure unit tests pass locally after every commit (`git rebase -i main
      --exec 'just rapid-tests'`)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Add backport label to latest release (format: 'backport release-branch-name')

### CODE REVIEW

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        test-type: [ 'test', 'everest-models-test', 'doc']
+        test-type: [ 'test', 'everest-models-test', 'everviz-test','doc']
```
Comment (berland) [inline .github/workflows/build_and_test.yml]: [nitpick] Missing space separator after comma

File: .github/workflows/test_everest.yml [hunk 2]
```diff
@@ -78,6 +78,16 @@ jobs:
         uv pip install git+https://github.com/equinor/everest-models.git
         ERT_PYTEST_ARGS='--cov=ert --cov=everest --cov=_ert --cov-report=xml:cov1.xml --junit-xml=junit.xml -o junit_family=legacy -m everest_models_test' uv run just everest-tests
 
+    - name: Run tests requiring everviz
+      if: inputs.test-type == 'everviz-test'
+      run: |
+        uv pip install dash==2.17.1 # https://github.com/equinor/komodo-releases/issues/6431
```
Comment (berland) [inline .github/workflows/test_everest.yml]: Since ert is an open repo, and komodo-releases is a closed repo, the reason for pinning should probably be excplicitly stated here.

Comment (berland) [reply .github/workflows/test_everest.yml]: Actually, this pin should not be needed in this workflow, it only affect a limited set of users running onprem with corner-case setups.

File: tests/everest/entry_points/test_visualization_entry.py [hunk 3]
```diff
@@ -24,11 +25,32 @@ def test_expected_message_when_no_visualisation_plugin_is_installed(
     change_to_tmpdir, cached_example, min_config
 ):
     config_path, config_file, _, _ = cached_example("math_func/config_advanced.yml")
-    config = EverestConfig.load_file(Path(config_path) / config_file)
+    #config = EverestConfig.load_file(Path(config_path) / config_file)
```
Comment (berland) [inline tests/everest/entry_points/test_visualization_entry.py]: delete line ?

File: tests/everest/entry_points/test_visualization_entry.py [hunk 4]
```diff
@@ -24,11 +25,32 @@ def test_expected_message_when_no_visualisation_plugin_is_installed(
     change_to_tmpdir, cached_example, min_config
 ):
     config_path, config_file, _, _ = cached_example("math_func/config_advanced.yml")
```
Comment (berland) [inline tests/everest/entry_points/test_visualization_entry.py]: can this line also go away?

Comment (berland) [reply tests/everest/entry_points/test_visualization_entry.py]: probably not

File: tests/everest/entry_points/test_visualization_entry.py [hunk 5]
```diff
@@ -24,11 +25,32 @@ def test_expected_message_when_no_visualisation_plugin_is_installed(
...
     config_file = "test.yml"
     config = EverestConfig(**min_config)
     config.dump(config_file)
     with capture_streams() as (out, _):
         visualization_entry([config_file])
     assert "No visualization plugin installed!" in out.getvalue()
+
+
+@pytest.mark.skipif(
+    "GITHUB_ACTIONS" not in os.environ,
```
Comment (berland) [inline tests/everest/entry_points/test_visualization_entry.py]: If I install the plugin (how?) locally, then I would like to run this test, or not?

Comment (frode-aarstad) [reply tests/everest/entry_points/test_visualization_entry.py]: Added a skipif_no_everviz decorator

File: .github/workflows/test_everest.yml [hunk 6]
```diff
@@ -78,6 +78,14 @@ jobs:
         uv pip install git+https://github.com/equinor/everest-models.git
         ERT_PYTEST_ARGS='--cov=ert --cov=everest --cov=_ert --cov-report=xml:cov1.xml --junit-xml=junit.xml -o junit_family=legacy -m everest_models_test' uv run just everest-tests
 
+    - name: Run tests requiring everviz
+      if: inputs.test-type == 'everviz-test'
+      run: |
+        uv pip install dash==2.17.1 # https://github.com/plotly/dash/issues/3003
```
Comment (berland) [inline .github/workflows/test_everest.yml]: Was this necessary on Github action runners you said? (it works fine on my workstation without the pin)

Comment (frode-aarstad) [reply .github/workflows/test_everest.yml]: Looks like it can be removed now 👍🏻
